AppCenterReactNativeShared/ios/AppCenterReactNativeShared/AppCenterReactNativeShared.m

```diff
@@ -5,6 +5,7 @@
 @implementation AppCenterReactNativeShared
 
 static NSString *appSecret;
+static BOOL startAutomatically;
 static MSWrapperSdk * wrapperSdk;
```

Comment:
nit - `MSWrapperSdk *wrapperSdk`

Reply:
Will include it in my next PR that I am working on as this is existing code ;)